src/SonarScanner.MSBuild.TFS/CoverageReportProcessorBase.cs

```diff
@@ -86,7 +86,7 @@ public bool ProcessCoverageReports(ILogger logger)
                 coverageReportPaths.Any() &&
                 config.GetSettingOrDefault(SonarProperties.VsCoverageXmlReportsPaths, true, null, logger) == null)
             {
-                File.AppendAllText(propertiesFilePath, $"{SonarProperties.VsCoverageXmlReportsPaths}={string.Join(",", coverageReportPaths.Select(c => c.Replace(@"\", @"\\")))}");
+                File.AppendAllText(propertiesFilePath, $"{Environment.NewLine}{SonarProperties.VsCoverageXmlReportsPaths}={string.Join(",", coverageReportPaths.Select(c => c.Replace(@"\", @"\\")))}");
```

Comment:
There is some repetition (like the escaping, the join, the newLine). Try to extract it to a static method.